.eslintrc.js

```diff
@@ -9,13 +9,7 @@ module.exports = {
     es6: true,
   },
   plugins: ["jest"],
-  extends: [
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "airbnb",
-    "prettier",
-    "prettier/react",
-  ],
+  extends: ["eslint:recommended", "plugin:react/recommended", "airbnb", "prettier"],
```

Comment:
intensional removal of `prettier/react`? 🙃

Reply:
It was! It's been merged into prettier, I removed it when I got an error, I think when I updated prettier.

Reply:
Oh cool! Makes sense 👍